common/src/test/java/com/revenuecat/purchases/common/HTTPClientVerificationTest.kt

```diff
@@ -176,7 +176,6 @@ class HTTPClientVerificationTest: BaseHTTPClientTest() {
             .setResponseCode(responseCode)
             .setHeader(HTTPResult.SIGNATURE_HEADER_NAME, "test-signature")
             .setHeader(HTTPResult.REQUEST_TIME_HEADER_NAME, 1234567890L)
-            .setHeader(HTTPResult.ETAG_HEADER_NAME, "test-etag")
```

Comment:
Remove this since it wasn't realistic. As in, if the response has a payload, it shouldn't have an etag.